include/projdefs.h

```diff
@@ -60,6 +60,10 @@ typedef void (* TaskFunction_t)( void * );
     #define configUSE_LIST_DATA_INTEGRITY_CHECK_BYTES    0
 #endif
 
+#ifndef configUSE_MINI_LIST_ITEM

```

Comment:
All but hese two list related configuration parameters get defaulted in FreeRTOS.h.  Is there a reason they need to be defaulted here instead?  Maybe there is a reason configUSE_LIST_DATA_INTEGRITY_CHECK_BYTES was already defaulted here, but it could have been a mistake.